postgres/pgdump.go

```diff
@@ -249,6 +249,13 @@ func processCreateStmt(conv *internal.Conv, n nodes.CreateStmt) {
 		logStmtError(conv, n, fmt.Errorf("can't get table name: %w", err))
 		return
 	}
+	if len(n.InhRelations.Items) > 0 {
+		// We currently skip the inherited tables in processing. so we log this
```

Comment:
Comments should be complete sentences. 
How about just:

		// Skip inherited tables.

Reply:
updated comment